Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM ubuntu:14.04
+MAINTAINER Elico Corp <info@elico-corp.com>
```

Comment:
Indeed contact@elico-corp.com is an option
Sébastien Maillard On 09/25/2015 06:16 PM, Eric Caudal wrote:
>
> In Dockerfile 
> <https://github.com/Elico-Corp/odoo-docker/pull/5#discussion_r40416510>:
>
> > @@ -0,0 +1,95 @@
> > +FROM ubuntu:14.04
> > +MAINTAINER Elico Corp <info@elico-corp.com>
>
> info@ doesnot exist (this is not an issue it is forwarded.
> should we keep it like this or us contact@ or create a docker@
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Elico-Corp/odoo-docker/pull/5/files#r40416510>.
>